tests/acceptance/features/lib/FilesPageElement/SharingDialog.php

```diff
@@ -472,6 +475,32 @@ public function openPublicShareTab(Session $session) {
 		return $publicLinkTab;
 	}
 
+	/**
+	 * @param Session $session
+	 * @param boolean $cancel
+	 *
+	 * @return void
+	 */
+	public function removePublicLink(Session $session, $cancel = null) {
```

Comment:
Should not the default be `false`? 
```suggestion
	public function removePublicLink(Session $session, $cancel = false) {
```